CHANGELOG.md

```diff
@@ -19,6 +28,23 @@
 ### Bug fixes
 - Fixed indexing queue templates escaping.
 
+## 3.16.1
+
+### Updates
```

Comment:
@damcou I realized we missed notating some updates on the original change log - which I've added here in Github:  https://github.com/algolia/algoliasearch-magento-2/releases/tag/3.16.1 

I'm not sure if we'll have a 3.16.2 but we should be sure to update the 3.17 release I would think?

Reply:
I updated the changelog so it reflects the changelog from the 3.16.1 release :)